src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const errorMessage = 'Bzzz... Error!';
+  let count = 0;
+
+  const robot = (a) => {
```

Comment:
Can skip creating the variable and return it right away.
```suggestion
  return (a) => {
```